.gitignore

```diff
@@ -11,4 +11,8 @@ packages
 docfx/api
 
 #git merge files
-*.orig
\ No newline at end of file
+*.orig
```

Comment:
Why is this change in this PR? This is already in master. You need to rebase.

Reply:
Ups I made something wrong, sorry.